tests/data/mxnet_mnist/MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
It seems that some files on this PR, including this one, should be there. Maybe is it a result of a bad merge?

Please remove these files.

Reply:
Removed